0.17.0-release-notes.md

```diff
@@ -66,3 +66,7 @@ New features in the standard library:
 
 -   The `store:` module now exposes all functionalities of Elvish's persistent
     store.
+
+-   New `compare` command to allow efficient comparison of numbers, strings, and
+    lists ([#1347](https://b.elv.sh/1347)). This is the default comparator used
+    by `order`.
```

Comment:
This is inaccurate - `order` does not use a comparator, it uses a less-than callback. Better to say that this is consistent with the criteria used by `order`.

Reply:
I am also unsure what the "efficient" part refers to so I'll just remove it. If there's a specific reason to keep it I can add it back, probably with a different wording.